googletest/cmake/internal_utils.cmake

```diff
@@ -78,10 +78,11 @@ macro(config_compiler_and_linker)
     # http://stackoverflow.com/questions/3232669 explains the issue.
     set(cxx_base_flags "${cxx_base_flags} -wd4702")
   elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    set(cxx_base_flags "-Wall -Wshadow -Werror -Wno-error=sign-conversion")
+    set(cxx_base_flags "-Wall -Wshadow -Werror -Wno-error=sign-conversion -std=c++11 -frtti")
```

Comment:
Passing `-frtti` just reads weird.

RTTI is enabled by default. There is no need to specify it.

Reply:
> Passing `-frtti` just reads weird.
> 
> RTTI is enabled by default. There is no need to specify it.

What if a downstream project disables it, for some specific reason though? This is an attempt to be defensive by specifying the value explicitly.